src/games/gameitem.py

```diff
@@ -285,9 +290,13 @@ def update(self, message, old_client=None):
         # Spawn announcers: IF we had a gamestate change, show replay and hosting announcements
         if oldstate != self.state:
             if self.state == "playing":  # The delay is there because we have a 5 minutes delay in the livereplay server
-                QtCore.QTimer.singleShot(5*60000, self.announceReplay)
+                self.launched_at = message['launched_at']
+                if time.time() - self.launched_at > 5*60:  # most already running games at client start
+                    self.announceReplay()
+                else:
+                    QtCore.QTimer().singleShot(5*60100 - 1000*(time.time() - self.launched_at), self.announceReplay)
```

Comment:
Yes, it's strange to have the central gamelist in QListWIdgetItem in a QListWidge (with most Items hidden).
Better to have a Game and Games class, like Player and Players. (like you've done in #709, I would prefer it in client folder, but if we move player/players to model[s] folder too maybe .. cause 'model' doesn't sound like games, so I wouldn't look there for it .. naming logic ... hard stuff ;-) 
I just recently noted, that the server message is handed over from Widget to Widget (game replay coop). 
In the end The GamesWidget should only hold the hosted games.